indico/util/date_time.py

```diff
@@ -315,57 +315,93 @@ def _format_pretty_datetime(dt, locale, tzinfo, formats):
     return _format_datetime(dt, fmt, tzinfo, locale)
 
 
-def format_pretty_date(dt, locale=None, tzinfo=None):
+def format_pretty_date(dt, locale=None, tzinfo=None, capitalize=True):
     """Format a date in a pretty way using relative units if possible.
 
     :param dt: a date or datetime object. if a date is provided, its
                time is assumed to be midnight
     :param locale: the locale to use for formatting
     :param tzinfo: the timezone to use
+    :param capitalize: if the return string should be capitalized
     """
     if not isinstance(dt, datetime):
         dt = datetime.combine(dt, dt_time())
-    return _format_pretty_datetime(dt, locale, tzinfo, {
+    if capitalize:
+        return _format_pretty_datetime(dt, locale, tzinfo, {
+            # i18n: keep the single quotes around the string
+            'last_day': _("'Yesterday'"),
+            # i18n: keep the single quotes around the string
+            'same_day': _("'Today'"),
+            # i18n: keep the single quotes around the string
+            'next_day': _("'Tomorrow'"),
+            # i18n: keep the single quotes around the string, and only translate the "Last".
+            # i18n: EEEE is a babel-style placeholder for the long weekday (you should probably keep this as-is)
+            'last_week': _("'Last' EEEE"),
+            # i18n: EEEE is a babel-style placeholder for the long weekday (you should probably keep this as-is)
+            'next_week': _('EEEE'),
+            'other': '{date_fmt}'
+        })
+    else:
+        return _format_pretty_datetime(dt, locale, tzinfo, {
         # i18n: keep the single quotes around the string
-        'last_day': _("'Yesterday'"),
+        'last_day': _("'yesterday'"),
         # i18n: keep the single quotes around the string
-        'same_day': _("'Today'"),
+        'same_day': _("'today'"),
         # i18n: keep the single quotes around the string
-        'next_day': _("'Tomorrow'"),
+        'next_day': _("'tomorrow'"),
         # i18n: keep the single quotes around the string, and only translate the "Last".
         # i18n: EEEE is a babel-style placeholder for the long weekday (you should probably keep this as-is)
-        'last_week': _("'Last' EEEE"),
+        'last_week': _("'last' EEEE"),
         # i18n: EEEE is a babel-style placeholder for the long weekday (you should probably keep this as-is)
         'next_week': _('EEEE'),
         'other': '{date_fmt}'
     })
 
 
-def format_pretty_datetime(dt, locale=None, tzinfo=None):
+def format_pretty_datetime(dt, locale=None, tzinfo=None, capitalize=True):
```

Comment:
Same here. Consider keeping the code in one single block with inline checks for `capitalize`.